### DISCUSSION TITLE
Any way to use third party controls ?

### DISCUSSION THREAD

**Discussion by joer33304** [AvaloniaUI/Avalonia#6628] [Q&A] [answered]
I would like to port my Balzor app to avalonia but I rely heavily on third party controls. Most vendors have controls for many platforms, does any work with Avalonia ? (using syncfusion which has Balzor and UWP controls)

**Comment by rstm-sf** [comment, marked as answer]
No, they won't work. They need to be ported by creating a request, for example https://github.com/AvaloniaUI/Avalonia/discussions/6255

Also take a look at the [Libraries & Extensions](https://github.com/AvaloniaCommunity/awesome-avalonia#libraries--extensions) section from Awesome-Avalonia

**Comment by maxkatz6** [comment]
Alternatively, some controls can be used with interop layer. 
I.e. you can use NativeControlHost to have winforms/wpf controls. But it will be platform depended.
Or you can use any web-browser controls to have blazor/other js components.
